src/js/Slider.js

```diff
@@ -289,9 +299,11 @@ const SliderComponent = (
       onRNCSliderSlidingStart={onSlidingStartEvent}
       onRNCSliderSlidingComplete={onSlidingCompleteEvent}
       onRNCSliderValueChange={onValueChangeEvent}
-      enabled={!props.disabled}
+      enabled={!_disabled}
```

Comment:
I think we should remove the `enabled` prop completely (probably in another PR though).
Currently it introduces confusion and can lead to mistakes/errors.